packages/frontend/src/components/DashboardTiles/TileForms/TileUpdateChartTitle.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Modal, Stack, TextInput } from '@mantine/core';
+import { useForm } from '@mantine/form';
+
+interface TileUpdateModalProps {
+    title: string;
+    isOpen: boolean;
+    onClose: () => void;
+    onConfirm: (title: string) => void;
+}
+
+const TileUpdateModal = ({
+    title,
+    isOpen,
+    onClose,
+    onConfirm,
+}: TileUpdateModalProps) => {
+    const form = useForm({
+        initialValues: {
+            title,
+        },
+    });
+
+    const handleOnSubmit = form.onSubmit(({ title: newTitle }) => {
+        onConfirm(newTitle);
+        onClose();
+    });
+    return (
+        <Modal
+            size="lg"
+            opened={isOpen}
+            className="non-draggable"
+            onClose={onClose}
+            title="Edit chart title"
```

Comment:
you can add `centered` prop to Mantime modals to center them vertically

Reply:
This is not needed anymore   https://github.com/lightdash/lightdash/commit/5eab5b10827666304a432f5475e9b4d373becd08